dist/vespa.spec

```diff
@@ -437,6 +437,28 @@ Requires: %{name}-base-libs = %{version}-%{release}
 
 Vespa - The open big data serving engine - tools
 
+%package ann-benchmark
```

Comment:
This seems very specific. Do we expect to add other benchmarks as well ? In this case we should consider just having vespa-benchmarks as a package.